spec/vm_spec.rb

```diff
@@ -28,6 +28,50 @@
 
       subject.run_command("ls")
     end
+
+    it "captures the stderr by default" do
+      expect(command_runner).to receive(:run) do |command, opts|
+        expect(command).to eq("foo")
+        expect(opts[:stderr]).to be_a(StringIO)
+      end
+
+      subject.run_command("foo")
+    end
+
+    it "allows for setting a custom stderr behavior" do
+      expect(command_runner).to receive(:run).with("foo", stderr: :capture)
+
+      subject.run_command("foo", stderr: :capture)
+    end
+
+    context "with stderr output" do
+      before(:each) do
+        expect(command_runner).to receive(:run) do |command, opts|
```

Comment:
Unused block argument - `command`. If it's necessary, use `_` or `_command` as an argument name to indicate that it won't be used.